lsr-role2collection.py

```diff
@@ -0,0 +1,443 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# (c) 2020 Matt Martz <matt@sivel.net>
+# GNU General Public License v3.0+
+#     (see https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# Usage:
+# lsr-role2collection.py [--namespace NAMESPACE]
+#                        [--collection COLLECTION]
+#                        --src-path SRC_PATH
+#                        --dest-path DEST_PATH
+#                        --role ROLE
+#                        [-h]
+
+import argparse
+import os
+import re
+import fnmatch
+from shutil import copytree, copy2, ignore_patterns
+
+from pathlib import Path
```

Comment:
I think these imports are ok - they are pretty standard and should be available in el7 `python3-something` packages.